docs/Testing.TestingAndDebugging.md

```diff
@@ -61,7 +61,7 @@ The response then contains the **searchQueryStr** attribute, as follows:
     				"parent" : "/permid.org"
       			}
     		} ],
-    "searchQueryStr" : "{\n  \"from\" : 0,\n  \"size\" : 1,\n  \"query\" : {\n\"filtered\" : {\n  \"query\" : {\n\"bool\" : {\n  \"must\" : {\n\"match\" : {\n  \"CommonName.mdaas\" : {\n\"query\" : \"Disney\",\n\"type\" : \"phrase\"\n  }\n}\n  }\n}\n  },\n  \"filter\" : {\n\"bool\" : {\n  \"must\" : [ {\n\"term\" : {\n  \"parent\" : \"/permid.org\"\n}\n  }, {\n\"range\" : {\n  \"lastModified\" : {\n\"from\" : null,\n\"to\" : null,\n\"include_lower\" : true,\n\"include_upper\" : true\n  }\n}\n  } ]\n}\n  }\n}\n  },\n  \"fields\" : [ \"type\", \"path.system\", \"uuid.system\", \"lastModified.system\", \"length.content\", \"mimeType.content\", \"linkTo\", \"linkType\", \"dc.system\", \"indexTime.system\", \"quad.system\" ],\n  \"sort\" : [ {\n\"lastModified.system\" : {\n  \"order\" : \"desc\"\n}\n  } ]\n}"
+        "searchQueryStr" : "{\n  \"from\" : 0,\n  \"size\" : 1,\n  \"query\" : {\n\"filtered\" : {\n  \"query\" : {\n\"bool\" : {\n  \"must\" : {\n\"match\" : {\n  \"CommonName.mdaas\" : {\n\"query\" : \"Disney\",\n\"type\" : \"phrase\"\n  }\n}\n  }\n}\n  },\n  \"filter\" : {\n\"bool\" : {\n  \"must\" : [ {\n\"term\" : {\n  \"parent\" : \"/permid.org\"\n}\n  }, {\n\"range\" : {\n  \"lastModified\" : {\n\"from\" : null,\n\"to\" : null,\n\"include_lower\" : true,\n\"include_upper\" : true\n  }\n}\n  } ]\n}\n  }\n}\n  },\n  \"fields\" : [ \"type\", \"system.path\", \"system.uuid\", \"system.lastModified\", \"content.length\", \"content.mimeType\", \"linkTo\", \"linkType\", \"system.dc\", \"system.indexTime\", \"system.quad\" ],\n  \"sort\" : [ {\n\"system.lastModified\" : {\n  \"order\" : \"desc\"\n}\n  } ]\n}"
```

Comment:
it still shows some weird stuff like `CommonName.mdaas`.
tried the query locally. this is how it should look like:
```
"{\n  \"from\" : 0,\n  \"size\" : 1,\n  \"query\" : {\n    \"filtered\" : {\n      \"query\" : {\n        \"bool\" : {\n          \"must\" : {\n            \"match\" : {\n              \"fields.-Jamjg.CommonName\" : {\n                \"query\" : \"Disney\",\n                \"type\" : \"phrase\"\n              }\n            }\n          }\n        }\n      },\n      \"filter\" : {\n        \"bool\" : {\n          \"must\" : [ {\n            \"term\" : {\n              \"system.current\" : true\n            }\n          }, {\n            \"term\" : {\n              \"parent\" : \"/permid.org\"\n            }\n          } ],\n          \"must_not\" : {\n            \"term\" : {\n              \"system.kind\" : \"DeletedInfoton\"\n            }\n          }\n        }\n      }\n    }\n  },\n  \"fields\" : [ \"system.kind\", \"system.path\", \"system.uuid\", \"system.lastModified\", \"content.length\", \"content.mimeType\", \"link.to\", \"link.kind\", \"system.dc\", \"system.indexTime\", \"system.quad\", \"system.current\" ],\n  \"sort\" : [ {\n    \"system.lastModified\" : {\n      \"order\" : \"desc\"\n    }\n  } ]\n}"
```

P.S. if you want to document debug-info, a better format is `yaml` as it preserves the internal json without escaping.
```
GET <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Disney&debug-info&format=yaml&pretty&length=1&override-mimetype=text/plain
pagination:
    next: http://cm-well-prod.int.thomsonreuters.com/permid.org?op=search&format=yaml&from=2018-02-06T03%3A03%3A15.335Z&to=2018-02-06T03%3A03%3A15.335Z&qp=CommonName.mdaas%3ADisney&length=1&offset=1
    last: http://cm-well-prod.int.thomsonreuters.com/permid.org?op=search&format=yaml&from=2018-02-06T03%3A03%3A15.335Z&to=2018-02-06T03%3A03%3A15.335Z&qp=CommonName.mdaas%3ADisney&length=1&offset=15428
    self: http://cm-well-prod.int.thomsonreuters.com/permid.org?op=search&format=yaml&from=2018-02-06T03%3A03%3A15.335Z&to=2018-02-06T03%3A03%3A15.335Z&qp=CommonName.mdaas%3ADisney&length=1&offset=0
    type: PaginationInfo
    first: http://cm-well-prod.int.thomsonreuters.com/permid.org?op=search&format=yaml&from=2018-02-06T03%3A03%3A15.335Z&to=2018-02-06T03%3A03%3A15.335Z&qp=CommonName.mdaas%3ADisney&length=1&offset=0
type: SearchResponse
results:
    fromDate: &id001 !!org.joda.time.DateTime {}
    total: 15428
    offset: 0
    toDate: *id001
    infotons:
    -   system:
            path: /permid.org/1-21635762683
            indexTime: 1517886195433
            parent: /permid.org
            dataCenter: dc1
            lastModified: *id001
            uuid: 126ece9b9ac2fe698a510a8df633921e
        type: ObjectInfoton
    length: 1
    type: SearchResults
    searchQueryStr: |-
        {
          "from" : 0,
          "size" : 1,
          "query" : {
            "filtered" : {
              "query" : {
                "bool" : {
                  "must" : {
                    "match" : {
                      "fields.-Jamjg.CommonName" : {
                        "query" : "Disney",
                        "type" : "phrase"
                      }
                    }
                  }
                }
              },
              "filter" : {
                "bool" : {
                  "must" : [ {
                    "term" : {
                      "system.current" : true
                    }
                  }, {
                    "term" : {
                      "parent" : "/permid.org"
                    }
                  } ],
                  "must_not" : {
                    "term" : {
                      "system.kind" : "DeletedInfoton"
                    }
                  }
                }
              }
            }
          },
          "fields" : [ "system.kind", "system.path", "system.uuid", "system.lastModified", "content.length", "content.mimeType", "link.to", "link.kind", "system.dc", "system.indexTime", "system.quad", "system.current" ],
          "sort" : [ {
            "system.lastModified" : {
              "order" : "desc"
            }
          } ]
        }
```

Reply:
Fixed (reran with yaml formatting).